src/layouts/components/Header.tsx

```diff
@@ -496,7 +538,7 @@ const drawerMenuItemLinks = [
   //   pathname: "/consortium/asset_breakdown"
   // },
   {
-    text: "On-Chain Governance",
+    text: "Governance",
```

Comment:
do we want to change the order in mobile menu too?

Reply:
oh yes! Changed